vendor.conf

```diff
@@ -14,7 +14,7 @@ github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76 # v1.1.0
 github.com/dgrijalva/jwt-go a2c85815a77d0f951e33ba4db5ae93629a1530af
 github.com/docker/distribution 0d3efadf0154c2b8a4e7b6621fff9809655cc580
 github.com/docker/docker 827cb09f87964ed38b46502f22a585f2ed4a78e1
-github.com/docker/compose-on-kubernetes 356b2919c496f7e988f6e0dfe7e67d919602e14e # master w/ v1alpha3+pullsecrets+pull-policy
+github.com/docker/compose-on-kubernetes e1f7354ed0de256b6ebf5bfa5a90a76460036c52
```

Comment:
Should we align this with a tagged version? See the discussion on https://github.com/docker/cli/pull/1757#issuecomment-476229818